holoviews/operation/datashader.py

```diff
@@ -68,74 +69,91 @@ def dataset_pipeline(dataset, schema, canvas, glyph, summary):
 
 
 class Aggregate(ElementOperation):
+    """
+    Aggregate implements 2D binning for any valid HoloViews Element
+    type using datashader. By default it will simply count the number
+    of values in each bin but custom aggregators can be supplied
+    implementing mean, max, min and other reduction operations.
```

Comment:
I wouldn't call them "custom"; they are just other aggregators.  So maybe "bin but other aggregators can be supplied explicitly".